apps/music/models.py

```diff
@@ -5,37 +5,6 @@
 User = get_user_model()
 
 
-class Genre(models.Model):
-    POP = "pop"
-    COUNTRY = "country"
-    RNB = "rnb"
```

Comment:
My biggest change was making the Genre a static set of choices instead of a model. It looks like they don't change as often so we can get rid of a JOIN on most queries. Adding a new genre should also be pretty easy, and a custom migration could be created to update/delete genres if that's ever needed